docs/plugins/plugin-development-guide/naming.rst

```diff
@@ -0,0 +1,59 @@
+Naming changes
```

Comment:
Just a general thought: it would be great to have some way to achieve it automatically (e.g. only pass your vendor plus plugin name and it would rename all the things and delete exemplary stuff).

Reply:
Totally agreed 👍 However right now it has to be done manually, so I'm documenting it anyway